backend/internal/controllers/user.go

```diff
@@ -0,0 +1,59 @@
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/TritonSE/words-alive/internal/database"
+	"github.com/TritonSE/words-alive/internal/models"
+)
+
+type UserController struct {
+	// Hold a reference to the database so that all of the functions below can use it
+	// Keeps an authenticator to authenticate token
```

Comment:
Not anymore it doesn't, delete this? :)

Reply:
And actually I think the comment above is unnecessary too, this is a common pattern that's pretty self-documenting